packages/core/src/features/cohort/availableCohortsSlice.ts

```diff
@@ -582,6 +587,14 @@ export const selectCurrentCohortId = (state: CoreState): string | undefined =>
 export const selectCohortMessage = (state: CoreState): string[] | undefined =>
   state.cohort.availableCohorts.message;
 
+/**
+ * Returns the current cohort login status
```

Comment:
+1

Reply:
I will update it. Good call.

Reply:
Something like: 

The case composition of a cohort can be different based on the login status of the user. The login status is attached to the cohort so that the portal and applications are updated are when the login state changes. 

Feel free to use any or none of this.

Reply:
This is much better than what I came up with. Too bad I didn't notice in time. I will still update it.

Reply:
Okay this has been updated. Thank you 😃